### PR TITLE
Update Porfolio with picture, facts, and paragraph

### PR DESCRIPTION
**Pull request by aionashby**
This is just a preliminary request to ensure that I am doing all of this right!

### CODE REVIEW

**Comment by aionashby** [comment]
> This is just a preliminary request to ensure that I am doing all of this right!

Final change was made! I think I am done, at least for now!

**Comment by aionashby** [comment]
Final version of my portfolio! https://aashby-sps-summer20.uc.r.appspot.com/ - that's the link to it!

**Review by pwilthew — APPROVED** [review]

**Comment by pwilthew** [comment]
Great!  Hey! I'm also a Rewriting the Code Fellow.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -5,14 +5,192 @@
...
+    <div id="fact-container"></div>
+  </div>
+  <div class="main">
+    <h2>Projects</h2>
+    <h5>Some projects I have worked on include:</h5>
+    <p>
+        <ul>
+            <li>Space Invaders - My own interpretation of the game Space Invaders!</li>
+            <li>Duck Runner - A game that involves guiding a duck through obstacles</li>
+            <li>Plumber Escape - An applicaiton that focused on strengthening my skills in algorithms like
```
Comment (pwilthew) [inline portfolio/src/main/webapp/index.html:161]: Oops, typo in "application"

Comment (aionashby) [reply portfolio/src/main/webapp/index.html:161]: Thanks I will fix that as I work on Week 2!

**Comment by aionashby** [comment]
> Great! Hey! I'm also a Rewriting the Code Fellow.

OMG what a small world! That is so cool!

**Pull request merged**